### ISSUE TITLE
Set up mailing list (i.e., "let me know closer to the event")

### ISSUE THREAD

**Issue by dcwalk** [ournetworks/2018#19]

**Comment by dcwalk** [comment]
@garrying -- here is how I did a form the last time I did a form: https://github.com/dcwalk/walkingsidewalkcity/blob/master/index.html#L72

**Comment by dcwalk** [comment]
Hey @garrying -- is this something we could aim to get up on the website this weekend (along with CFP)?

**Comment by garrying** [comment]
@dcwalk Yeah, we should aim to have it on the website this weekend. Getting CFP up this weekend sounds like a good goal.

**Comment by dcwalk** [comment]
@garrying has a branch [`sign-up-form`](https://github.com/ournetworks/ournetworks.ca/tree/sign-up-form) and will PR and ping people for review! Targeting end of weekend Feb 26 launch
